tutorials/5-docker-intro/index.md

```diff
@@ -0,0 +1,288 @@
+---
+parent:
+  title: Docker Introduction
+  description: Get started with Docker
+  number: 
+tags:
+  - tutorial
+  - dev-ops
+order: 0
+title: Docker introduction
+---
+
+<div class="tm-overline tm-rf-1 tm-lh-title tm-medium tm-muted">Know your way around Docker</div>
+<h1 class="mt-4 mb-6">Docker Introduction</h1>
+
+[Docker](https://www.docker.com) is a kind of virtualization, with performance benefits.
```

Comment:
@xavierlepretre Can we qualify "virtualization"?

"Docker is a kind of XXXXXXXX virtualization with performance benefits."

Reply:
Yes